devnet/README.md

```diff
@@ -0,0 +1,71 @@
+# Gossamer Devnet
+
+Docker container and Docker Compose for a Gossamer Devnet currently with three authority nodes running the `gssmr` chain with keys `alice`, `bob` and `charlie`.
+
+## Requirements
+
+- Docker Compose (https://docs.docker.com/desktop/mac/install/)
+
+## Setup
+
+The following files need to be on the root of the Gossamer repo to work.
+- `alice.Dockerfile`
+- `bob.Dockerfile`
+- `docker-compose.yml`
+
+To symlink to the repository root
+```
+# cd into your Gossamer root
+cd ~/dev/gossamer
+
+ln -s devnet/alice.Dockerfile alice.Dockerfile
+ln -s devnet/bob.Dockerfile bob.Dockerfile
+ln -s docker-compose.yml docker-compose.yml
```

Comment:
This doesn't seem right (creating link at the same path). Also, root already has a `docker-compose.yml`. Is this a mistake or am I missing something?

Reply:
Good catch.  Updated the README.